README.md

```diff
@@ -0,0 +1,32 @@
+# React-Frontend Plugin Template 
+
+Reference example for using [decky-frontend-lib](https://github.com/SteamDeckHomebrew/decky-frontend-lib) in a [PluginLoader](https://github.com/SteamDeckHomebrew/PluginLoader) plugin.
+
+## PluginLoader Discord [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/ZU74G2NJzk)
+
+## Developers
+
+### Dependencies
+
+This template relies on the user having `pnpm` installed on their system.  
+This can be downloaded from `npm` itself which is recommended. 
+
+#### Linux
+```bash
+sudo npm i -g pnpm
+```
+
+### Installation
+
+1. Clone the repository to use as an example for making your plugin.
+2. In your clone of the repository run these commands:
+   1. ``pnpm i``
+   2.  (Optional, needed to solve build errors)
```

Comment:
This is misleading, IMO should just be under some useful commands/notes section for how to update the plugin library version.

Reply:
> This is misleading, IMO should just be under some useful commands/notes section for how to update the plugin library version.

Addressed in latest commit, changed from "Installation" to "Keeping Your Plugin Updated" and dropped usage of "clone" etc.

Reply:
Did not mean change installation, meant to move the part about updating the lib to somewhere separate. Installation should really be "Getting started" or something like that.

Reply:
> Did not mean change installation, meant to move the part about updating the lib to somewhere separate. Installation should really be "Getting started" or something like that.

Ah gotcha. The suggested change thing is always a couple lines up for you so if you want mobile to stop janking you out probably best to go a few lines down from what you want changed. 👍🏻